src/parser.c

```diff
@@ -689,6 +689,81 @@ json_object *parser_device_json_create(const char *device_id,
 	return device;
 }
 
+static json_object *schema_item_create_obj(knot_msg_schema *schema)
```

Comment:
About commit message: f5fb030401e6d6ca09cdab7ce5e7e5a9bc79b415
... and rewrite *them*.

Reply:
Done.